src/Microsoft.AspNet.Mvc.Formatters.Xml/XmlDataContractSerializerOutputFormatter.cs

```diff
@@ -148,17 +148,20 @@ protected virtual DataContractSerializer CreateSerializer(Type type)
         }
 
         /// <summary>
-        /// Creates a new instance of <see cref="XmlWriter"/> using the given stream and the <see cref="WriterSettings"/>.
+        /// Creates a new instance of <see cref="XmlWriter"/> using the given <see cref="TextWriter"/> and
+        /// <see cref="WriterSettings"/>.
         /// </summary>
-        /// <param name="writeStream">The stream on which the XmlWriter should operate on.</param>
+        /// <param name="writer">
+        /// The underlying <see cref="TextWriter"/> which the <see cref="XmlWriter"/> should write to.
+        /// </param>
         /// <returns>A new instance of <see cref="XmlWriter"/></returns>
         public virtual XmlWriter CreateXmlWriter(
-            Stream writeStream,
+            TextWriter writer,
```

Comment:
Made this change so we could do the 'nested using' pattern. We don't want to have problems if the user throws here and doesn't dispose the writer.